docker-support/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM fluxcd/website:hugo-0.78.2-extended
```

Comment:
Where is the Dockerfile for this image?

Reply:
It's in the `hugo` directory, which is a clone at the particular version we need

See this part of the Makefile:
```
hugo:
        git clone https://github.com/gohugoio/hugo.git -b v$(HUGO_VERSION)

.PHONY: docker-support
docker-support: hugo
        docker build -t $(DEV_IMAGE_NAME) --build-arg HUGO_BUILD_TAGS=extended hugo/
```

We would not normally need to do this, but Hugo doesn't publish their own images, with or without the `HUGO_BUILD_TAGS=extended` arg set. (I believe I got the specific instructions about how Hugo had to be built from `@dholbach`, via `@juozasg`)

The image is built by the `docker-support` target, I don't think it needs to change very often, I envisioned that one of us would build it and push it once, and there is no immediate need for automation around this part.

Reply:
When I tested this, all the images were of the format `kingdonb/fluxcd-website:hugo-version` etc. I figured that wasn't appropriate to publish in the fluxcd org.

I changed it here, but someone with access to the fluxcd credentials will have to build and publish these two support images just once before this can be merged.

Reply:
I can build it and push it to Docker Hub if you want.

Reply:
I added this information to the README.

Reply:
Nice, can we push the image to make it easier to try this out?

Reply:
:+1: